api/public/cerbos/request/v1/request.proto

```diff
@@ -651,6 +651,23 @@ message InspectPoliciesRequest {
     (google.api.field_behavior) = OPTIONAL,
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Filter policies by version with regexp"}
   ];
+  repeated string id = 5 [
+    (google.api.field_behavior) = OPTIONAL,
+    (buf.validate.field).repeated = {
+      unique: true,
+      max_items: 25,
+      items {
+        string {
+          min_len: 1,
+          max_len: 1280
+        }
```

Comment:
I don't think that uniqueness is strictly required because we can deal with that on the server side. It'd be a bad experience to reject a request because the user accidentally included the same ID twice.

Do we have a limit of `max_len = 1280` defined elsewhere? Otherwise, I don't think we need to limit the maximum length.

```suggestion
      max_items: 50,
      items {
        string {
          min_len: 1,
        }
```

Reply:
`GetPolicy` request also has a limit of 1280.

Reply:
OK. I think we can remove that restriction because it doesn't include the scope nor the periods and slashes that are part of the ID.